libs/npm/core/src/components/box/Box.tsx

```diff
@@ -16,7 +16,7 @@ export const Box: React.FC<IBoxProps> = ({ isLoading, className, children, ...re
     <styled.Box className={`box${className ? ` ${className}` : ''}`} {...rest}>
       {isLoading && (
```

Comment:
Validate that changing the parent component will result in the child state being updated.

Reply:
I just changed the loading component here. The rest of the logic should work as before.